test/integration/driver.go

```diff
@@ -83,16 +82,14 @@ func init() {
 		"kube config file or empty for in-cluster")
 	flag.StringVarP(&hub, "hub", "h", "gcr.io/istio-testing",
 		"Docker hub")
-	flag.StringVarP(&tag, "tag", "t", "",
-		"Docker tag (default <username>_YYYMMDD_HHMMSS formatted date)")
+	flag.StringVarP(&tag, "tag", "t", "test",
+		"Docker tag")
```

Comment:
I removed the default value and made the user specify tag explicitly to avoid mismatch with bin/e2e.sh. PTAL